### PR TITLE
[WIP]add msvc in cc.py

### PR DESCRIPTION
**Pull request by yajiedesign**
The compiler is successful. Wait until I do some testing
https://github.com/dmlc/tvm/issues/524

- [x] llvm create_shared
- [x] llvm export_library
- [ ] llvm -system-lib create_shared
- [ ] llvm -system-lib export_library

### CODE REVIEW

**Comment by tqchen** [comment]
@yajiedesign please let me know when you finished the testing

**Comment by yajiedesign** [comment]
testing fail.I'm still trying to figure it out.

**Comment by tqchen** [comment]
I can see a major obstacle here could be the dependency on the dll content on TVM runtime. We have avoided that by passing in function pointers back into the load dll https://github.com/dmlc/tvm/blob/master/src/runtime/dso_module.cc#L55

So when we build without --system-lib mode, things should not depend on libtvm or libtvm_runtime. The --system-lib mode is intended to bundle the library together with libtvm.dll and other parts. It is not necessary for windows since dll loading is not banned, so we do not have to support this case

**Comment by tqchen** [comment]
another problem on the cuda export (the generated c code) could due to the code generated variable not being explicitly marked as dll export, which hides their symbols. This won't happen in linux

**Comment by yajiedesign** [comment]
ok done.wait ci

File: python/tvm/contrib/cc.py [hunk 1]
```diff
@@ -24,26 +29,92 @@ def create_shared(output,
...
+                flags = flags.replace(",DATA", "").split(' ')
+                flags = [item for item in filter(lambda x: x != '', flags)]
+                link_cmd += flags
+
+    # tvm_runtime = find_lib_path("tvm_runtime.lib")
+    # link_cmd += [tvm_runtime]
+    link_cmd += ["-EXPORT:__tvm_main__"]
+    link_cmd += [temp_path + "dllmain.obj"]
+    link_cmd += ["-out:" + output]
+    print(link_cmd)
```
Comment (tqchen) [inline python/tvm/contrib/cc.py]: avoid print

File: tests/python/unittest/test_module_load.py [hunk 2]
```diff
@@ -1,7 +1,15 @@
-import tvm
+
+try:
+    import tvm
```
Comment (tqchen) [inline tests/python/unittest/test_module_load.py]: avoid doing this, always set PYTHONPATH before testing

**Pull request merged**